### PR TITLE
Updated participant file and added finalist file.

### PR DESCRIPTION
**Pull request by cmcodes1**
@gkanishk 
I have completed the challenge and here's my PR for participants and finalist file.

### CODE REVIEW

File: content/finalist/ttjc@finalist@cmcodes1/index.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+---
+title: cmcodes
+type: "finalist"
+name: "Chandra Mohan Pandey"
+portfolio: "https://cmcodes1.github.io"
+github-link: "https://github.com/cmcodes1"
+linkedin-link: "https://linkedin.com/in/cmcodes"
+looking-for: "job"
+description: "Analyst | Tech Geek | Developer"
+image-link: "https://avatars2.githubusercontent.com/u/34341671?s=460&u=73e18d14ad731e271dcb5a16e215cc7cf626721b&v=4"
```
Comment (gkanishk) [inline content/finalist/ttjc@finalist@cmcodes1/index.md:10]: Please name the variables as follows:
```
github_link:
linkedin_link:
looking_for:
image_link:

```

File: content/finalist/ttjc@finalist@cmcodes1/index.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+My projects:
+
+#### Studify
+
+_description_ A student utility android application.
+
+_stack_ React Native
+
+_github link_ https://github.com/cmcodes1/Studify
```
Comment (gkanishk) [inline content/finalist/ttjc@finalist@cmcodes1/index.md:56]: Any deployment link or for preview provide release link.

Comment (cmcodes1) [reply content/finalist/ttjc@finalist@cmcodes1/index.md:56]: It has already been pushed to Google Play Console, but will be available in 6 days, after they review it. So, I can't provide any link for that yet! Will update it when it gets approved from Play Store. Although, I do have attached the screenshots of the app in the README.md file of the mentioned GitHub link.

File: content/finalist/ttjc@finalist@cmcodes1/index.md (hunk 1 above)
Comment (cmcodes1) [reply content/finalist/ttjc@finalist@cmcodes1/index.md]: @gkanishk Done. Please check #1127.

File: content/finalist/ttjc@finalist@cmcodes1/index.md (hunk 2 above)
Comment (gkanishk) [reply content/finalist/ttjc@finalist@cmcodes1/index.md:56]: provide any link to download the apk at present later update when it gets approved

**Comment by gkanishk** [comment]
Awesome!! 
Kindly provide details here #1033

**Comment by cmcodes1** [comment]
Done! Thanks for all your inputs! 😊

**Pull request merged**